### PR TITLE
Renamed notebook, EPT path is dynamic from CMR query now. TODO: pdal

### PR DESCRIPTION
**Pull request by wildintellect**
The main goal is updating the EPT example to work with the Ops env (GCC).
* Used a CMR query with the maap api to get the granule, extracting the url to the ept.json
* Renamed the notebook to remove testing from the name.

TODO:
* Noting this seems to be missing the WFS example, can't recall if that's because we didn't add it to the maap api yet?
* Blocked on testing without a working PDAL install in GCC ADE notebook. Going to try and solve before moving PR out of draft.

### CODE REVIEW

**Comment by abarciauskas-bgse** [comment]
Unfortunately I don't think the features example got added, but I was able to track it down:
https://github.com/MAAP-Project/maap-documentation/blob/abarciauskas-bgse_v1d2-demo/docs/source/query/query-ept-stores.ipynb

**Comment by wildintellect** [comment]
I was aware that the Features example wasn't in there yet. I forget why we kept it out of the docs, were we waiting on having a api domain name for it?

**Comment by wildintellect** [comment]
We decided TODO are for another ticket/PR.

File: docs/source/query/ept_stores.ipynb [hunk 1]
```diff
@@ -20,29 +22,31 @@
...
-    "This notebook provides sample code for querying and visualizing the Entwine Point Tiles Store for ATL08 V003.\n",
+    "import gpstime\n",
+    "from pyproj import CRS, Transformer\n",
+    "import folium\n",
+    "import geopandas as gpd\n",
+    "#import pdal\n",
     "\n",
-    "[Entwine Point Tiles (EPT)](https://entwine.io/entwine-point-tile.html) are a new \"cloud-optimized\" storage format for large point cloud data."
+    "from maap.maap import MAAP\n",
+    "maap = MAAP(maap_host='api.ops.maap-project.org')"
```
Comment (abarciauskas-bgse) [inline docs/source/query/ept_stores.ipynb:49]: Is there a reason why this is `api.ops.maap-project` instead of just `api.maap-project` ?

Comment (wildintellect) [reply docs/source/query/ept_stores.ipynb:49]: https://github.com/MAAP-Project/ZenHub/issues/367

**Review by abarciauskas-bgse — APPROVED** [review]

**Comment by abarciauskas-bgse** [comment]
@wildintellect I ran this and the search works great, thanks

```
##Get the ept store url from a CMR search
atl08_global_ept = maap.searchGranule(concept_id="G1200352824-NASA_MAAP", limit=10)

## Define the global EPT source for our data queries
ept_store = atl08_global_ept[0].getDownloadUrl()
print(ept_store)
```

**Pull request merged**